seeders/20200128023056-demo-user.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('users', [
```

Comment:
what's the difference between a seeder and inserting an entry via migration?

Reply:
We may have integrity issues if we have migrations that modify tables where previous migrations already seeded data. If we separate them, we can re-seed freely without having to redo single migrations (which can be dangerous if there are more recent dependent migrations)

Also on prod/test envs, we might need to migrate without seeding